mock/src/lib.rs

```diff
@@ -69,7 +70,7 @@ impl MockRequestDispatcher {
 impl DispatchSignedRequest for MockRequestDispatcher {
     type Future = FutureResult<HttpResponse, HttpDispatchError>;
 
-    fn dispatch(&self, request: SignedRequest) -> Self::Future {
+    fn dispatch(&self, request: SignedRequest, _timeout: Option<StdDuration>) -> Self::Future {
```

Comment:
I assume we'll actually use this timeout when getting closer to being done? 😉

Reply:
What would you envision the mock to do with the timeout?